commit db671becfe0f2826dac7f479d091d5a23feccb47
Author:     Hao Lin <hao.lin@samsung.com>
AuthorDate: Mon Oct 12 14:13:32 2026 +0800
Commit:     Hao Lin <hao.lin@samsung.com>
CommitDate: Mon Oct 12 14:13:32 2026 +0800

    Revert lld default linker in Tizen-Base-Toolchain
    
    The lld default-linker change (-fuse-ld=lld and -Wl,--undefined-version
    in build_ldflags, lld in Support/Keep) has not passed validation yet.
    Remove it from Tizen-Base-Toolchain so that the next build uses the
    GNU linker as before. It will be submitted again after validation.
    
    This reverts commit bae4ebdd732e3c01611de2414434d4c5c162763c.
    
    Change-Id: I0b3f8e75e3664d9e044eeba972d7d4669c59a342
    Signed-off-by: Hao Lin <hao.lin@samsung.com>

diff --git a/TIZEN/Tizen/Tizen-Base-Toolchain/project_config b/TIZEN/Tizen/Tizen-Base-Toolchain/project_config
index 333eabeb6..e267b054c 100644
--- a/TIZEN/Tizen/Tizen-Base-Toolchain/project_config
+++ b/TIZEN/Tizen/Tizen-Base-Toolchain/project_config
@@ -350,5 +350,5 @@ Macros:
 :Macros
 %endif
 Macros:
-%build_ldflags -fuse-ld=lld -Wl,-z,relro,--as-needed%{?_ldflags_arch_extra}
+%build_ldflags -Wl,-z,relro,--as-needed%{?_ldflags_arch_extra}
 :Macros
